packages/utils/src/lib/reports/formatting.ts

```diff
@@ -58,3 +66,58 @@ export function metaDescription(
   }
   return '';
 }
+
+/**
+ * Link to local source for IDE
+ * @param source
+ * @param reportLocation
+ *
+ * @example
+ * linkToLocalSourceInIde({ file: '/src/index.ts'}, {outputDir: '/.code-pushup'}) // [/src/index.ts](../src/index.ts)
+ */
+export function linkToLocalSourceForIde(
+  source: SourceFileLocation,
+  options?: Pick<MdReportOptions, 'outputDir'>,
+): InlineText {
+  const { file, position } = source;
+  const { outputDir } = options ?? {};
+  const unixPath = toUnixPath(file);
+
+  // NOT linkable
+  if (!outputDir) {
+    return md.code(unixPath);
+  }
+
+  const relativePath = pathPosix.relative(outputDir, unixPath);
```

Comment:
Why use `posix.relative` instead of `relative`?

Reply:
I tried out several other configurations, and none of those using regular `path.relative()` could normalize Windows path correctly

Reply:
Oh, so it's to prevent conversion to Windows paths? OK, then POSIX makes sense.